src/GraphQL.Serialization.Newtonsoft/Http/DocumentWriter.cs

```diff
@@ -0,0 +1,49 @@
+using System.Buffers;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace GraphQL.Http
+{
+    public class DocumentWriter : IDocumentWriter
```

Comment:
Please look at #1322

Reply:
Adding the comment here too, I like the removal of `Http` since it isn't really Http related.

Reply:
This should now be moved.